### DISCUSSION TITLE
Wails V3有计划支持拦截webview的所有请求吗？

### DISCUSSION THREAD

**Discussion by ho0xi** [wailsapp/wails#3083] [Q&A]
Wails V3有计划支持拦截webview的所有请求吗？类似于Electron的Debug webRequest？

**Comment by JoiLa** [comment]
:)   我认为这种不安全的功能，Wails v3暂时不会出来的。不过你可以自己 fork 这个项目魔改下。

**Reply by ho0xi** [reply]
我看您也提过相关的问题或者有拦截所有请求的需求，您是如何实现的？

**Reply by JoiLa** [reply]
![image](https://github.com/wailsapp/wails/assets/42708003/58958c0b-a4cb-4f47-9e57-4252f993bfc2)

主要是在这边。在 访问 asset 基础之上，可以自己 fork 改下。本地开发请参考 [link](https://wails.io/zh-Hans/docs/guides/local-development)

**Reply by ho0xi** [reply]
ok
